### ISSUE TITLE
Gtk windows work in Repl - but not in Juno?

### ISSUE THREAD

**Issue by hustf** [JuliaGraphics/Gtk.jl#165]
Thanks for the good work! Coming just recently from VB6 and Fortran, I now run Julia 0.3.11 on Windows 64 bit, and also on a Windows 32 bit. 

Having applied both the two fixes below, Gtk seems to work well from the Repl. Refer to:
https://github.com/JuliaLang/Gtk.jl/blob/master/doc/installation.md
https://github.com/JuliaLang/WinRPM.jl ('Usage example' to guide WinRPM in choosing a Gtk3 variant)

Juno runs the same Julia.exe file as Repl does, and I confirm that juliarc.jl is run at startup through 'println("..juliarc.jl..")

Windows created as below are displayed and can be moved, but not resized. They won't display anything, and they don't 'Respond' to Windows. They can still be closed from Juno / Gtk. So some package tests will run without discovering this. I hope for an easy way out, while not out of Juno?

Juno input / output:

```
    using Gtk
    win = Gtk.@Window("My window")
    showall(win)
    println(win)
    GtkWindowLeaf(name="", parent, width-request=-1, height-request=-1, visible=TRUE,        sensitive=TRUE, app-paintable=FALSE, can-focus=FALSE, has-focus=FALSE, is-focus=FALSE, can-default=FALSE, has-default=FALSE, receives-default=FALSE, composite-child=FALSE, style, events=0, no-show-all=FALSE, has-tooltip=FALSE, tooltip-markup=NULL, tooltip-text=NULL, window, opacity=1,000000, double-buffered, halign=GTK_ALIGN_FILL, valign=GTK_ALIGN_FILL, margin-left, margin-right, margin-start=0, margin-end=0, margin-top=0, margin-bottom=0, margin=0, hexpand=FALSE, vexpand=FALSE, hexpand-set=FALSE, vexpand-set=FALSE, expand=FALSE, scale-factor=1, border-width=0, resize-mode, child, type=GTK_WINDOW_TOPLEVEL, title="My window", role=NULL, resizable=TRUE, modal=FALSE, window-position=GTK_WIN_POS_NONE, default-width=-1, default-height=-1, destroy-with-parent=FALSE, hide-titlebar-when-maximized=FALSE, icon, icon-name=NULL, screen, type-hint=GDK_WINDOW_TYPE_HINT_NORMAL, skip-taskbar-hint=FALSE, skip-pager-hint=FALSE, urgency-hint=FALSE, accept-focus=TRUE, focus-on-map=TRUE, decorated=TRUE, deletable=TRUE, gravity=GDK_GRAVITY_NORTH_WEST, transient-for, attached-to, has-resize-grip, resize-grip-visible, application, is-active=FALSE, has-toplevel-focus=FALSE, startup-id, mnemonics-visible=FALSE, focus-visible=FALSE, is-maximized=FALSE)</addr>
```

**Comment by hustf** [comment]
To @vtjnash 's question at #147, the same packages are reported as installed both through Juno and through Repl. I

julia> Pkg.installed("Gtk")
v"0.8.3"
julia> Pkg.installed("WinRPM")
v"0.1.12"

**Comment by hustf** [comment]
Similar issues arise with Tk, GLPlot and other packages I tried. 
It seems Julia needs to be the boss of IDEs, because Windows don't talk face to face with serving applications.

This has been a very time consuming obstacle to me, so far a humble Julia noob. I propose a tedious workaround, but future IDEs ought perhaps to adapt  when running on windows.  Sorry for the IDE noise just here, keep up the work on Julia 0.4!

1) Start Julia Repl
2) Start Juno. 
3) Evaluation -> Kill Julia
4) Ctrl + space -> Connect to running session
5) Repl command: Paste string from clipboard with process info: using Jewel; @async Jewel.server(49438, 90)
6) Now the code above works both from Juno and Repl.

**Comment by vtjnash** [comment]
may be fixed by #161

**Comment by hustf** [comment]
Yes, thank you! I see no difference when run from Juno Atom Shell or from the REPL. Another bug prevented full operation, but this also seems to be fixed after https://github.com/JuliaLang/Gtk.jl/pull/204 . But that currently requires a checkout.
